CONTRIBUTING.md

```diff
@@ -25,58 +23,56 @@ Skills with the following would be beneficial:
 - Docker
 - CI/CD workflows
 
-Our latest task board can be found [here](https://github.com/orgs/hotosm/projects/22).
+Our latest task board is available [here](https://github.com/orgs/hotosm/projects/22).
 
 ## Report bugs and suggest improvements
 
 The [issue queue](https://github.com/hotosm/fmtm/issues) is the best way to get started. There are issue templates for BUGs and FEATURES that you can use, you could also create your own. Once you have submitted an issue, it will be assigned one label from the following [label categories](https://github.com/hotosm/fmtm/labels). If you are wondering where to start, you can filter by the **good first issue label**.
 
 ## :handshake: Thank you
 
-Thank you very much in advance for your contributions!! Please ensure you refer to our **Code of Conduct**.
-If you've read the guidelines, but are still not sure how to contribute on Github, please reach out to us via our Slack **#geospatial-tech-and-innovation**.
+We're grateful for your contributions. Please review our **Code of Conduct**.
+If you have any questions about contributing on GitHub, don't hesitate to reach out to us via our Slack channel, **#geospatial-tech-and-innovation**.
 
 # Code Contribution guidelines
 
 ## Workflow
 
-We operate the "Fork & Pull" model explained at [About Pull
-Requests](https://help.github.com/articles/about-pull-requests/)
+We follow the "Fork & Pull" model for contributions it is explained at [About Pull Requests](https://help.github.com/articles/about-pull-requests/). Here's how it works:
+
+Fork the Project: Fork the project into your own repository.
+
+Create a Topic Branch: Create a topic branch in your repository to work on your contribution.
+
+Make Pull Requests: Create one or more pull requests back to the main repository.
```

Comment:
Add the sentence: `Complete PRs are preferred, but Draft PRs are also accepted if the feature/update is large.`